charts/any-resource/values.yaml

```diff
@@ -0,0 +1,8 @@
+anyResources:
+  exampleResource: |-
```

Comment:
each time I use this chart I have to override this value ... can we comment tihs out or move it to an example file?

Reply:
If nothing is specified, the chart becomes invalid as it will contain no resources.
But instead of this example in values.yaml, I could create an `else` in the `any-resource.yaml` which will render the example if no resource is specified.